lib/rubocop/cop/lint/symbol_conversion.rb

```diff
@@ -21,10 +27,49 @@ module Lint
       #   :underscored_symbol
       #   :'hyphenated-string'
       #
+      # @example EnforcedStyle: strict
```

Comment:
Can you mark `(default)` to the EnforcedStyle?

```suggestion
      # @example EnforcedStyle: strict (default)
```

Reply:
Yep! Sorry, I forgot to.

Reply:
Fixed.